bin/varnishd/cache/cache_hash.c

```diff
@@ -355,6 +355,42 @@ hsh_insert_busyobj(const struct worker *wrk, struct objhead *oh)
 	return (oc);
 }
 
+/*---------------------------------------------------------------------
+*/
+
+static unsigned
+hsh_rush_match(struct req *req)
+{
+	struct objhead *oh;
+	struct objcore *oc;
+	const uint8_t *vary;
+
+	oc = req->objcore;
+	CHECK_OBJ_ORNULL(oc, OBJCORE_MAGIC);
```

Comment:
This should be CHECK_OBJ_NOTNULL(). Callers should not call it if `req->objcore == NULL`. That makes it obvious at the call sites that this function needs an OC to operate.